### DISCUSSION TITLE
对多行公式识别率偏低，以及有 hallucination 问题

### DISCUSSION THREAD

**Discussion by kkew3** [OleehyO/TexTeller#38] [feedback]
非常感谢你的工作！拿 https://gregorygundersen.com/blog/2019/12/23/random-fourier-features/ 里的若干公式试了试，单行公式总体还可以，但多行公式大概 30--40% 预测的公式有语法错误、或有行缺失，我尝试将多行公式拆成单行来截图，会好很多。另外即使是单行公式也有时有 hallucination 问题，例如

![hallucination](https://github.com/OleehyO/TexTeller/assets/13264071/e843ed53-7ee0-4d64-8bc8-f4a36370d46d)

$\mathbb R^D$ 变成了 $\mathbb S^2$。用的 `NUM_BEAM=4`。

**Comment by OleehyO** [comment]
感谢您的反馈和测试。

目前这个版本的TexTeller在某些情况下确实还存在问题（一些比较少见的符号、矩阵、大型的多行公式），总体来看可能还是因为这些场景在训练数据里没有cover到，这些问题应该能在第二版得到解决。
